zanata-war/src/main/webapp/WEB-INF/urlrewrite.xml

```diff
@@ -196,6 +196,18 @@
     <to last="true">/profile/merge_account.seam$1</to>
   </rule>
 
+  <rule>
+    <from casesensitive="true">^/profile/view/([^/]+)(/([^?]*)\??(.*))?$</from>
```

Comment:
it's like project page. /profile/view/someusername will go to someone's profile page with username 'someusername'.